src/api/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ z3_add_component(api
     api_context.cpp
     api_datalog.cpp
     api_datatype.cpp
+    api_debug.cpp
```

Comment:
I will change this to be exception by default for all API use, exceptiions will be enabled on context creation time.